src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
It's not recommended to modify the `__proto__` object directly because it can lead to performance issues and is generally considered bad practice. Instead, you should use `Object.defineProperty` to define `sort2` on the `Array.prototype`.